app/forms/journeys/further_education_payments/computing_courses_form.rb

```diff
@@ -42,8 +42,8 @@ def checkbox_options
             name: course_option_description("tlevel_digitalproduction")
           ),
           OpenStruct.new(
-            id: "ib_certificate_compsci",
-            name: course_option_description("ib_certificate_compsci")
+            id: "ibo_level3",
```

Comment:
i think its worth suffixing `_compsci` or something similar to that respect, otherwise downstream for reporting and data consumers may find it harder to figure out what was selected